platform/kube/controller.go

```diff
@@ -568,6 +569,34 @@ func (c *Controller) HostInstances(addrs map[string]bool) []*model.ServiceInstan
 	return out
 }
 
+// TODO error indicator
+func (c *Controller) GetIstioServiceAccounts(hostname string) []string {
```

Comment:
When does this method get called? Does it also get triggered when some pods are deleted?